client/epics/err-epic.js

```diff
@@ -1,10 +1,16 @@
 import { makeToast } from '../../common/app/Toasts/redux';
 
-export default function errorSaga(actions) {
+export default function errorEpic(actions) {
   return actions.filter(({ error }) => !!error)
     .map(({ error }) => error)
     .doOnNext(error => console.error(error))
-    .map(() => makeToast({
-      message: 'Something went wrong, please try again later'
-    }));
+    .map((error) => {
+      // Get data according to normalized ajax error.
+      const data = error.xhr.responseText ? JSON.parse(error.xhr.responseText)
```

Comment:
This will throw if the error does not have an xhr message.

Also, this couples the error epic to xhr messages. Instead the error object should be normalized at the source, not here.